### PR TITLE
fix(instances): indentation json MTA-1991

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

**Review by RoRoJ — APPROVED** [review]

File: compute/instances/api-cli/migrating-gp-to-pro.mdx [hunk 1]
```diff
@@ -98,17 +98,17 @@ Local and block snapshots have different formats. To support the migration from
...
+  {
+    "task": {
+        "id": "400f529a-faf9-4171-bcba-172118713d15",
+        "description": "server_backup",
+        "status": "pending",
+        "href_from": "/servers/4295c408-aaad-456a-95a0-7f5f6c2f2354/action",
+        "href_result": "/images/34d12fd2-6792-4dcb-81d0-b3dd1a3ddc29",
+        "started_at": "2022-05-11T10:43:40.777057+00:00",
+        "terminated_at": null
+            }
```
Comment (yrapaport-scw) [inline compute/instances/api-cli/migrating-gp-to-pro.mdx:110]:
```suggestion
    "task": {
        "id": "400f529a-faf9-4171-bcba-172118713d15",
        "description": "server_backup",
        "status": "pending",
        "href_from": "/servers/4295c408-aaad-456a-95a0-7f5f6c2f2354/action",
        "href_result": "/images/34d12fd2-6792-4dcb-81d0-b3dd1a3ddc29",
        "started_at": "2022-05-11T10:43:40.777057+00:00",
        "terminated_at": null
    }
```

File: compute/instances/api-cli/migrating-gp-to-pro.mdx [hunk 2]
```diff
@@ -126,42 +126,41 @@ Local and block snapshots have different formats. To support the migration from
...
-    }
+  "server": {
+      "id": "998b88d1-1d71-4a0a-8432-5965222d14a1",
+      "name": "pro2-test",
+      "arch": "x86_64",
+      "commercial_type": "PRO2-S",
+      (...)
+      "state": "stopped",
+      (...)
+            }
```
Comment (yrapaport-scw) [inline compute/instances/api-cli/migrating-gp-to-pro.mdx:163]:
```suggestion
  "server": {
      "id": "998b88d1-1d71-4a0a-8432-5965222d14a1",
      "name": "pro2-test",
      "arch": "x86_64",
      "commercial_type": "PRO2-S",
      (...)
      "state": "stopped",
      (...)
  }
```

File: compute/instances/api-cli/migrating-gp-to-pro.mdx [hunk 3]
```diff
@@ -172,21 +171,21 @@ Local and block snapshots have different formats. To support the migration from
     --header 'X-Auth-Token: '$SCW_SECRET_KEY'' \
     --data-raw '{
       "action": "poweron"
-    }'
+  }'
```
Comment (yrapaport-scw) [inline compute/instances/api-cli/migrating-gp-to-pro.mdx:174]: The bracket should be aligned with --data-raw
The problem is with the curl command above

File: compute/instances/api-cli/migrating-gp-to-pro.mdx [hunk 4]
```diff
@@ -172,21 +171,21 @@ Local and block snapshots have different formats. To support the migration from
...
+  {
+    "task": {
+        "id": "d7902607-2fe0-45f7-874f-72201e2654d4",
+        "description": "server_batch_poweron",
+        "status": "pending",
+        "href_from": "/servers/998b88d1-1d71-4a0a-8432-5965222d14a1/action",
+        "href_result": "/servers/998b88d1-1d71-4a0a-8432-5965222d14a1",
+        "started_at": "2022-05-11T10:52:23.222827+00:00",
+        "terminated_at": null
+        }
```
Comment (yrapaport-scw) [inline compute/instances/api-cli/migrating-gp-to-pro.mdx:187]:
```suggestion
    "task": {
        "id": "d7902607-2fe0-45f7-874f-72201e2654d4",
        "description": "server_batch_poweron",
        "status": "pending",
        "href_from": "/servers/998b88d1-1d71-4a0a-8432-5965222d14a1/action",
        "href_result": "/servers/998b88d1-1d71-4a0a-8432-5965222d14a1",
        "started_at": "2022-05-11T10:52:23.222827+00:00",
        "terminated_at": null
    }
```

File: compute/instances/api-cli/migrating-gp-to-pro.mdx [hunk 5]
```diff
@@ -196,15 +195,15 @@ Local and block snapshots have different formats. To support the migration from
...
-    }
+  {
+  "server": {
+      "id": "998b88d1-1d71-4a0a-8432-5965222d14a1",
+      "name": "pro2-test",
+      (...)
+      "state": "running",
+      (...)
+            }
+  }
```
Comment (yrapaport-scw) [inline compute/instances/api-cli/migrating-gp-to-pro.mdx:206]:
```suggestion
  {
    "server": {
      "id": "998b88d1-1d71-4a0a-8432-5965222d14a1",
      "name": "pro2-test",
      (...)
      "state": "running",
      (...)
    }
  }
```

**Review by yrapaport-scw — CHANGES_REQUESTED** [review]

**Pull request merged**